src/components/Submarine/Submarine.jsx

```diff
@@ -0,0 +1,235 @@
+import _ from 'lodash';
+import React from 'react';
+import { lucidClassNames } from '../../util/style-helpers';
+import { createClass, filterTypes, findTypes, omitProps } from '../../util/component-types';
+import * as reducers from './Submarine.reducers';
+import SplitHorizontal from '../SplitHorizontal/SplitHorizontal';
+import ChevronIcon  from '../Icon/ChevronIcon/ChevronIcon';
+import GripperHorizontalIcon  from '../Icon/GripperHorizontalIcon/GripperHorizontalIcon';
+
+const cx = lucidClassNames.bind('&-Submarine');
+
+const {
+	any,
+	bool,
+	func,
+	node,
+	number,
+	string,
+	oneOf,
+	oneOfType,
+} = React.PropTypes;
+
+/**
+ * {"categories": ["layout"]}
+ *
+ * `Submarine` renders a collapsible, resizeable side bar panel next to primary content.
+ */
+const Submarine = createClass({
+	displayName: 'Submarine',
+
+	reducers,
+
+	propTypes: {
+		/**
+		 * Appended to the component-specific class names set on the root
+		 * element. Value is run through the `classnames` library.
+		 */
+		className: string,
+		/**
+		 * Direct children must be types {Submarine.Primary, Submarine.Bar, Submarine.Title}.
+		 * All content is composed as children of these respective elements.
+		 */
+		children: node,
+		/**
+		 * Sets the starting height of the Bar.
+		 */
+		height: oneOfType([number, string]),
+		/**
+		 * Force the Submarine to be expanded or collapsed.
+		 */
+		isExpanded: bool,
+		/**
+		 * Allows animated expand and collapse behavior.
+		 */
+		isAnimated: bool,
+		/**
+		 * Render the Submarine to the top or bottom of primary content.
+		 */
+		position: oneOf(['top', 'bottom']),
+		/**
+		 * Disable user resizing of the Submarine.
+		 */
+		isResizeDisabled: bool,
+		/**
+		 * Set the title of the Submarine. (alias for `Title` and `Submarine.Title`)
+		 */
+		title: any,
```

Comment:
I'm still not convinced we need these aliases since the capitalized child props are so prevalent in our library. What do you think @sodiumjoe ?